api/lib/infrastructure/repositories/assessment-result-repository.js

```diff
@@ -68,4 +90,36 @@ module.exports = {
 
     return bookshelfToDomainConverter.buildDomainObject(BookshelfAssessmentResult, latestAssessmentResultBookshelf);
   },
+
+  async get({ certificationCourseId }) {
+    const assessment = await knex('assessments')
+      .select('id')
+      .where({ certificationCourseId })
+      .orderBy('createdAt', 'desc')
+      .first();
+
+    if (assessment) {
+      const assessmentId = assessment.id;
+
+      const latestAssessmentResult = await knex('assessment-results')
+        .select('assessment-results.*')
```

Comment:
Comme suggéré par Jonathan
```suggestion
      const latestAssessmentResult = await knex('assessment-results').select()
```

ou encore 

```suggestion
      const latestAssessmentResult = await knex('assessment-results')
```